packages/tools/devtools/devtools-view/src/test/TelemetryView.test.tsx

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type DevtoolsFeatureFlags,
+	DevtoolsFeatures,
+} from "@fluidframework/devtools-core/internal";
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import { act } from "react-dom/test-utils";
```

Comment:
Is this the recommended import pattern? They don't surface test utils from the package root?

If this is the expected usage pattern, we should add an exception for this pattern to the package's eslint config directly IMO.

https://github.com/import-js/eslint-plugin-import/blob/6d34c88a91a9cb7556700b7cb83c8a27731ff302/docs/rules/no-internal-modules.md